packages/remark-lint-no-undefined-references/readme.md

```diff
@@ -33,15 +38,29 @@ No messages.
 ###### In
 
 ```markdown
-[bar][]
```

Comment:
I don't know why this example disappeared. I did not remove it from the source file.